docs/API/REST-V2-Usage.md

```diff
@@ -1213,10 +1213,8 @@ start as query parameters to support legacy behavior).
 | skip             | int    | Optional. Number of versions to skip.                                                                                                                                                                                                                                                                        |
 | limit            | int    | Optional. The number of versions to be returned per page of pagination. Defaults to 20.                                                                                                                                                                                                                      |
-| revision_start   | int    | Optional. The version order number to start at, for pagination.                                                                                                                                                                                                                                              |
-| revision_end     | int    | Optional. The version order number to end at, for pagination.                                                                                                                                                                                                                                                |
-| start_time_str   | string | Optional. Timestamp to start looking for applicable versions.                                                                                                                                                                                                                                                |
-| end_time_str     | string | Optional. Timestamp to stop looking for applicable versions.                                                                                                                                                                                                                                                 |
+| start   | int    | Optional. The version order number to start at for pagination. It will return anything less than the start.                                                                                                                                                                                                                                              |
+| revisions_after     | int    | Optional. Will return the version with that revision number and anything with a greater revision number (and therefore more recent).                                                                                                                                                                                                                                                 |
```

Comment:
start and revisions_after are different enough that I think this is pretty confusing, considering they do the same thing and both consider revisions, but yeah thinking of a name that complements start is hard. We could change the name of this to something else (but still support start since people are already using it, i.e. just assign start to our new variable name during "parse"), and then we could try to be more consistent with the names revision_start / revision_end used for modify (assuming they behave similarly).

Alternatively, the suggestion from the ticket to have a boolean flip directions for start might be clearest while maintaining backwards compatibility.

Reply:
(I realize there's a default version limit, I'm looking into that now)

Reply:
Please ignore the above. I'm looking into it some more and will comment when it's ready for re-review.

Reply:
Ready for re-review. I deprecated start and added `revision_start` and `revision_end`. I can't parse revision_start to start because one uses `lt` and one use `lte`.